### PR TITLE
Final edits to established vaccines

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Minor changes made during submission

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Review by agitter — APPROVED** [review]

**Review by RLordan — APPROVED** [review]
Thanks @rando2

**Comment by rando2** [comment]
Thank you both for review! I'm waiting for confirmation that this version got through screening, and then I'll merge!

**Comment by rando2** [comment]
News this morning is that the manuscript was accepted! Yay! So this can be our official version 2!

File: README.md
Comment (agitter) [inline README.md:19]:
```suggestion
| [Vaccines- Established Platforms](https://greenelab.github.io/covid19-review/#application-of-traditional-vaccine-development-strategies-to-sars-cov-2) | Application of Traditional Vaccine Development Strategies to SARS-CoV-2 | #830 | Accepted 🎉  | _mSystems_ | [Release](https://github.com/greenelab/covid19-review/releases/tag/vaccines-trad-v2.1); [Preprint](https://arxiv.org/abs/2208.08907) |
```
I'll call the corrected v2 release v2.1 so I don't have to delete my early v2. v2.1 will be the commit made after we merge this pull request.

**Comment by agitter** [comment]
@rando2 is it okay for me to merge this and tag v2.1 now?

**Comment by rando2** [comment]
Hi @agitter\-- Yes, I think that should be good!

**Comment by agitter** [comment]
@rando2 I tagged and released this version: https://github.com/greenelab/covid19-review/releases

Are you planning to post this updated version to arXiv as well? It does improve on v1 in many ways.

**Comment by rando2** [comment]
Thank you @agitter! Yes, I was planning to update both vaccines ms on arXiv -- I don't know if I should wait for proofs or just go for it!

**Comment by agitter** [comment]
Including novel vaccines in the updates is a good idea. We can fix the consortium author noted in https://github.com/greenelab/covid19-review/issues/1175#issuecomment-1285736319

**Pull request merged**
